### ISSUE TITLE
Publish plugin to Rider's plugin market

### ISSUE THREAD

**Issue by jinek** [ForNeVeR/AvaloniaRider#12]
**Notes (by @ForNeVeR):**
- [x] don't forget to update the build/maintenance instructions, and check the `publishPlugin.ps1` script

**Comment by ForNeVeR** [comment]
Yes, thanks, we're planning to publish it on the market after at least something will start to work :)

**Comment by ForNeVeR** [comment]
Currently, the publishing is planned to be done after all the tasks in the [First release](https://github.com/ForNeVeR/AvaloniaRider/projects/1) project are finished.

**Comment by ForNeVeR** [comment]
Alright, I've enabled automatic publishing of the preview builds (the channel is named "dev"). The plugin isn't yet available in the main Rider plugin marketplace; I've added [the instructions](https://github.com/ForNeVeR/AvaloniaRider/blob/HEAD/README.md#development-releases-may-be-unstable) to download and install the plugin from various sources, including the `dev` channel in the plugin marketplace, and GitHub directly.

Note that the dev repository isn't working yet (even while I've already pushed the first package to it); it will be automatically enabled in a couple of days, while the plugin is under the verification.

I'll still publish the plugin into the main repository (so it'll be available by default) after release 1.0. For now, consider the builds are only for the very brave souls.
